docs/datenlord_monitoring.md

```diff
@@ -49,7 +49,7 @@ Use the following default username and password to log in. Once you log in with
 User: admin
 Pass: admin
 ```
-Datenlord has already built a dashboard templates. To use Datenlord dashboard, just import [datanlord_dashboard.json](../scripts/datenlord-e2e-test.yaml) file in Grafana.
+Datenlord has already built a dashboard templates. To use Datenlord dashboard, just import [datanlord_dashboard.json](../scripts/setup/alertmanager-alerts.yaml) file in Grafana.
```

Comment:
Is this a mistake?

Reply:
Yes, the original `e2e-test` is a mistake, and `alerts` is as well.
I think the `Datenlord-Dashboard.json` is the right one.